### PR TITLE
feat(balancer-container): Auto detect relevant Balancer pool to query data from

### PR DESCRIPTION
**Pull request by nicholaspai**
Checks if user has selected a yToken against a list of yield tokens, default is the yUSD-SEP20 token.

Queries balancer pool data from yield token.

Also adds information about farming UMA during roll between SEPT and OCT

Preview prior to roll on august 28 23:00 UTC:
Screen Shot 2020-08-25 at 17 19 13

Preview after roll:
Screen Shot 2020-08-25 at 17 19 40

### CODE REVIEW

File: features/yield/Yield.tsx [hunk 1]
```diff
@@ -57,6 +57,14 @@ const Yield = () => {
             >
               here
             </a>
+            , and to learn about rolling between expiries see the article{" "}
```
Comment (chrismaree) [inline features/yield/Yield.tsx:60]: +1

**Review by chrismaree — APPROVED** [review]
this looks like a pretty good start! I think that you should hide the yield tab if you are on a non-yield token though. for example I can select ethbtc & still see a price, which is confusing. Once you've changed this I think we are good to go.

**Comment by nicholaspai** [comment]
> this looks like a pretty good start! I think that you should hide the yield tab if you are on a non-yield token though. for example I can select ethbtc & still see a price, which is confusing. Once you've changed this I think we are good to go.

Agreed with this. I implemented pls play around!

File: containers/Balancer.ts [hunk 2]
```diff
@@ -50,30 +73,38 @@ const useBalancer = () => {
...
   } = useQuery(TOKENS, {
+    skip: !selectedTokenAddress,
     context: { clientName: "BALANCER" },
-    variables: { tokenId: empAddress.toLowerCase() },
+    variables: { tokenId: selectedTokenAddress },
     pollInterval: 5000,
   });
   const { loading: poolLoading, error: poolError, data: poolData } = useQuery(
-    POOL(JSON.stringify(poolTokensList.map((token) => token.toLowerCase()))),
+    POOL(JSON.stringify(poolTokenList)),
```
Comment (nicholaspai) [inline containers/Balancer.ts:82]: I set these queried addresses explicitly to lower case below when I set state

**Comment by mrice32** [comment]
Don't forget to request reviewers!

**Comment by chrismaree** [comment]
This is great! you new changes are perfect. thanks for adding this @nicholaspai . I'm going to merge this on your behalf because people are asking about this feature in discord.

**Pull request merged**